Application/Ed-Fi-Ods.sln

```diff
@@ -1,405 +1,396 @@
-		{5AC84333-D566-41FE-8AD1-BDEDA5F1B0A6} = {26FBA24F-2F1E-47EA-BDE2-EF57AE81D65D}
-		{D0335B4F-B79B-4CCB-AB11-A4106C070513} = {DC40CF8E-5A05-454E-8D56-6157F3A25C82}
-		{06D25DDA-12C5-4EFA-AB3D-FDF08AE7A432} = {14352B5E-2434-4117-A063-2BBD4D87464D}
-		{3AD03A98-4A2C-42F1-997D-2468A0CCB77D} = {F014EF5F-7ACE-48B8-BF62-DDBC9AD90630}
-		{14352B5E-2434-4117-A063-2BBD4D87464D} = {F014EF5F-7ACE-48B8-BF62-DDBC9AD90630}
-		{6B56B650-F9DB-4037-9A0E-029560C4B2AD} = {26FBA24F-2F1E-47EA-BDE2-EF57AE81D65D}
-		{E3FD8373-9915-4E06-BA2B-9DCEA0DAD6BC} = {6B596494-5C35-4A04-93A8-13B21EBF9910}
-		{BA9A0F5B-FD66-4914-B6A1-0EF09C99465F} = {5A1504BF-181C-4948-B9A7-FAA57C88B087}
-		{8D60B22F-838D-44EC-9281-A088DBD53C91} = {5A1504BF-181C-4948-B9A7-FAA57C88B087}
-		{CFD55DFF-ED40-4495-85B2-BF9A0A43D879} = {6B596494-5C35-4A04-93A8-13B21EBF9910}
-		{3FA79048-ED88-42B9-8C32-4C51FD2AC312} = {3FF80F38-E44A-4CE0-B274-793E3E9B7042}
-		{FB19CCCC-DDD6-4751-83FF-937C39CEAA31} = {F014EF5F-7ACE-48B8-BF62-DDBC9AD90630}
-		{26FF26B8-8A79-474C-BFAE-AA853248151C} = {FB19CCCC-DDD6-4751-83FF-937C39CEAA31}
-		{BCD5C6A3-16D4-46CA-AD13-1F38A1A4245A} = {F25ED154-FB97-4F39-933B-808D1B3B6D3E}
-		{82F9555D-5653-4A36-ACAA-D8948EB22FAF} = {6B596494-5C35-4A04-93A8-13B21EBF9910}
-		{6710953A-5A70-4E95-A27C-1855FF492F52} = {F25ED154-FB97-4F39-933B-808D1B3B6D3E}
-		{61C8DC76-C6EF-43EA-8820-0796E6D3D53C} = {3AD03A98-4A2C-42F1-997D-2468A0CCB77D}
-		{DFD89624-8424-4878-8DF4-3C7C449AF990} = {E214F607-B637-4AF5-9A68-61CB86507E4A}
-		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A} = {3AD03A98-4A2C-42F1-997D-2468A0CCB77D}
-		{F77C0D3C-0CBC-45AF-A0C0-2604D629CF6E} = {DE7A9EA6-C3EE-4F6F-BEFB-3101E0D7DF5E}
-		{DDEE72C1-0AC6-43D9-8398-31E78746A4C4} = {26FBA24F-2F1E-47EA-BDE2-EF57AE81D65D}
-		{1ACF010B-2092-4377-844E-FD710B6404BD} = {4F8E0044-5DB6-4964-A8F2-3776560FB39E}
-		{E18A4602-21A7-4737-BEB3-D3ED06EB4C81} = {DC40CF8E-5A05-454E-8D56-6157F3A25C82}
-		{42E7C6B0-4EFF-4AB6-8083-7AF7CBE2288F} = {14352B5E-2434-4117-A063-2BBD4D87464D}
-		{D67B54A7-0275-484C-8F6C-28808383A0F6} = {3AD03A98-4A2C-42F1-997D-2468A0CCB77D}
-		{5D29E54C-0788-42D3-AC34-F7E144975458} = {14352B5E-2434-4117-A063-2BBD4D87464D}
-		{A90A9B8F-3939-49D1-80C4-D586A0EAF94F} = {14352B5E-2434-4117-A063-2BBD4D87464D}
-		{64381013-0EDC-46D3-9E24-E5500DF00A2A} = {E214F607-B637-4AF5-9A68-61CB86507E4A}
-		{81FE570F-FDA0-4184-8F26-D925423FDE57} = {DE7A9EA6-C3EE-4F6F-BEFB-3101E0D7DF5E}
-		{658A15C4-A69B-4825-BE4B-9B3F37FA7FA2} = {F25ED154-FB97-4F39-933B-808D1B3B6D3E}
-		{37CB77F7-1C51-480C-B536-CDBF5DE26E10} = {14352B5E-2434-4117-A063-2BBD4D87464D}
-		{11894549-6088-46EA-BBD9-52F254E70FDA} = {26FBA24F-2F1E-47EA-BDE2-EF57AE81D65D}
-		{D2E6E31D-FA21-4CF6-8C59-CF69BE4FDD50} = {26FBA24F-2F1E-47EA-BDE2-EF57AE81D65D}
-	EndGlobalSection
-	GlobalSection(ExtensibilityGlobals) = postSolution
-		SolutionGuid = {91F27EA2-CFF0-4054-A019-B8AA23AF9319}
-	EndGlobalSection
-EndGlobal
+Microsoft Visual Studio Solution File, Format Version 12.00

```

Comment:
curious why is this whole file changed?

Reply:
also are we ready to remove the old sandbox admin application?

Reply:
As far as I know, we are ready to remove sandboxAdmin.Web.